crates/dojo-indexer/README.md

```diff
@@ -30,13 +34,21 @@ Dojo Indexer is a command line tool that allows you to index data from a given D
 
 ## Usage
 
-To run the `dojo-indexer` command, open your terminal or command prompt, navigate to the directory containing the Dojo Indexer source code, and execute the following command:
+To run the `dojo-indexer` command, open your terminal or command prompt, make sure you are in the `dojo-indexer` directory.
+
+Export your parameters
+
+```
+export DOJO_WORLD=
+export APIBARA_NODE=
+export RPC_NODE=
+```
 
 ```
-cargo run --bin dojo-indexer <world> <node> <rpc> <database_url>
+cargo run --bin dojo-indexer DOJO_WORLD APIBARA_NODE RPC_NODE DATABASE_URL
```

Comment:
Hm I think we should avoid suggesting using ENV vars the passing them as args for now. In the future we could update the cli to read from the ENV directly and properly namespace them

Reply:
Clap has a [neat feature (with the `env` feature)](https://docs.rs/clap/latest/clap/_derive/#arg-attributes) where it accepts parameters both as arguments/flags and as env variables.